lib/Minify/JS/JShrink.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class Minify\JS\JShrink
+ *
+ * @package Minify
+ */
+
+namespace Minify\JS;
+
+/**
+ * Wrapper to Javascript Minifier built in PHP http://www.tedivm.com
+ *
+ * @package Minify
+ * @author  Elan Ruusamäe <glen@pld-linux.org>
+ * @link    https://github.com/tedious/JShrink
+ *
+ */
+class JShrink
+{
+    /**
+     * Contains the default options for minification. This array is merged with
+     * the one passed in by the user to create the request specific set of
+     * options (stored in the $options attribute).
+     *
+     * @var string[]
+     */
+    protected static $defaultOptions = array('flaggedComments' => true);
```

Comment:
Does the default operation preserve comments? If so I think we should just add usage to the cookbook.